spec/factory_bot/aliases_spec.rb

```diff
@@ -1,29 +1,31 @@
 describe FactoryBot, "aliases" do
-  context "aliases for an attribute" do
-    subject { FactoryBot.aliases_for(:test) }
-    it      { should include(:test) }
-    it      { should include(:test_id) }
+  it "for an attribute should include the original attribute and a version suffixed with '_id'" do
+    aliases = FactoryBot.aliases_for(:test)
+
+    expect(aliases).to include(:test)
+    expect(aliases).to include(:test_id)
```

Comment:
The thing I *don't* like about this is that, when assertions are combined like this, if one of them fails, you'd have to look up file name and line number from the backtrace to know which one it was (as opposed to just reading the test description at the CLI).  Thoughts?

Reply:
How about this?

```suggestion
    expect(aliases).to include(:test, :test_id)
```

Does that give a useful failure message?

Reply:
<img width="1037" alt="Screen Shot 2019-10-11 at 2 30 22 PM" src="https://user-images.githubusercontent.com/3839713/66686259-b5da9880-ec33-11e9-96ae-4f4740038e6c.png">


I feel like it does.  Anyone feel different?

Reply:
I changed it to `.not_to` for the purposes of generating a failure, but I'd imagine a `.to` failure would look similar.